tensorflow/tools/ci_build/windows/cpu/pip/build_tf_windows.sh

```diff
@@ -32,28 +32,11 @@ set -e
 # This script is under <repo_root>/tensorflow/tools/ci_build/windows/cpu/pip/
 # Change into repository root.
 script_dir=$(dirname $0)
-cd ${script_dir%%tensorflow/tools/ci_build/windows/cpu/pip}
+cd ${script_dir%%tensorflow/tools/ci_build/windows/cpu/pip}.
```

Comment:
Ha, that's because if we run this script under TF source root.
It would be 
`cd ${tensorflow/tools/ci_build/windows/cpu/pip%%tensorflow/tools/ci_build/windows/cpu/pip}`
, which is essentially `cd`, then it will just jump to your home directory.